src/api/link_post.js

```diff
@@ -0,0 +1,26 @@
+class LinkPostHandler {
+  constructor() {}
+
+  async handle(event, context, cb) {
+    //Parse body
+    let body;
+    try {
+      body = JSON.parse(event.body);
+    } catch (e) {
+      cb({ code: 403, message: "no json body: " + e.toString() });
+      return;
+    }
+
+    // Check if hash_token is present
+    if (!body.consent_signature || !body.linked_did) {
+      cb({ code: 403, message: "Missing data" });
+      return;
+    }
+
```

Comment:
The idea here is to extract the DID from the consent signature and link the did with the address. So later we can query the root hash either by the DID or the address